packages/flutter/lib/src/painting/placeholder_span.dart

```diff
@@ -41,6 +41,9 @@ abstract class PlaceholderSpan extends InlineSpan {
     TextStyle? style,
   }) : super(style: style);
 
+  /// The unicode character to represent a placeholder.
+  static const String placeholderCodeUnit = '\uFFFC';
```

Comment:
Is there a reason no to define this as an int code unit = 0xFFFC?
This way you could call buffer.writeCharCode() and avoid doing the slightly confusing placeholderCodeUnit.codeUnitAt(0) conversion in widget_span.dart

Reply:
Ok, I published #98971 as this PR has been merged already.